src/main/java/models/Deck.java

```diff
@@ -0,0 +1,37 @@
+package models;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Random;
+
+/**
+ * Created by Cameron on 2/18/2016.
+ */
+public class Deck {
+
+    public java.util.List<Card> cards = new ArrayList<>();
+
+    public Deck(){
+
+    }
+
+    public void buildDeck(){};
```

Comment:
Possibly reformat this class so that the brackets are placed similar to the class on line 14 (the previous class). Just that the formatting is consistent since they are both pretty much empty classes here.